### PR TITLE
Shoukou runaround

### LINKED ISSUE #1401: evac cant dock on shoko
## Description
it always uses arrivals even if evac is clear

### PR DESCRIPTION
**Pull request by ghost**
## About the PR

TLDR:
Makes a library, adds a librarian slot consequentially, moves epistemics all the way to where dorms was, redefines the chapel, obliterates all bathrooms since they were part of dorms (you now have to piss in maint botany or the bridge bathroom) and moves two of the dorms to the area by cryo.
Replaces Evac shuttle with the one it's intended to have, Kaeri, which fixes #1401 
Also does some cursed piping shit. I can't wait for velcro to destroy this in review!
## Why / Balance

A common complaint about Shoukou and an intent from the previous maintainer of it (Adrian) was that Epistemics blew up too much and you had to evac because it was in the middle of the damn station. This shifts Epistemics' front desk to the side. As a result:
- I had to move the reporter again (Little John the reporter keeps getting fucked over by command members taking his office, I swear to god, he never has peace) to the area by the chapel where Epistemics front was.
- The chapel expanded by one tile to the side.
- Botany expanded by one tile to the side.
- The library is a thing because I was trying to think of what Shoukou doesn't have that's in that space. Turns out it's the librarian.
- I decided to mow over the cryo room and put two of the missing dorms there and move cryo to the right. One day, I'll bring back the other four and the bathrooms, but for right now there's none, sorgy :(
- A few maints rooms got added (as a bonus,) because another common complaint with Shoukou is that it has not very many maints.
- Oh yeah, the golden toilet was mapped right in the area next to the bridge bar, where all thieves aren't going to suspect it

## Technical details

Map changes / Yaml.
## Media

May or may not be up to date. Most of these are progress screenshots and aren't indicative of what's actually reviewed.
Library.
image
Toilet botany.
image
Chapel (low quality, and may not be up to date because it was an image from before I made the library.)
image
Mystagogue's room.
image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Shoukou's Epistemics has moved and there is a new library where it was.

### CODE REVIEW

**Comment by ghost** [comment]
I realized I forgot the tiny fans thing so I'm now manifesting that this PR fails Build and Test Debug so that I can fix it before Velcro comes for me

**Comment by IamVelcroboy** [comment]
> I realized I forgot the tiny fans thing so I'm now manifesting that this PR fails Build and Test Debug so that I can fix it before Velcro comes for me

You have time. I'm going to sleepy land.

**Comment by ghost** [comment]
Okay, *NOW* it has everything and now I'm at your mercy, Velcro

**Comment by IamVelcroboy** [comment]
Tiny fans...
image
They should already get GPS units in their lockers. This seems like too many extras. Just keep one for backup or if LO wants to grab it
image
Check the cameras/Power units/fax machines for casing consistency (can edit easily in the file) A few "Unknown" fax machines.
image
image
image
I don't think this is linked
image
Light on door
image
easy fix
Screenshot 2024-09-14 214727

**Comment by IamVelcroboy** [comment]
A few redundant lights
remove middle one 
image
image
remove post
image
image

**Comment by IamVelcroboy** [comment]
Link this door
Screenshot 2024-09-14 215236

**Review by IamVelcroboy — APPROVED** [review]
Again, I'm going to merge because nothing is mechanically broken(I hope 🤞) Please make fixes above when you can.
